types/remark-heading-id/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@definitelytyped/no-outside-dependencies": "off"
```

Comment:
This should not be turned off.

Reply:
Removed the `.eslintrc` rule in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68755/commits/7bb4f6919d43df8052cd71c2012bb249805cb9ec